events/models.py

```diff
@@ -29,6 +29,7 @@ class Event(models.Model):
         default=True,
         help_text="If this is checked all users that attends the event needs to \
         be accepted by an admin.")
+    external_signup_url = models.CharField(max_length=80, blank=True)
```

Comment:
Yeah, I tried it out and it prepends the URL with the protocol if necessary which is pretty cool :)